app/controllers/posts_controller.rb

```diff
@@ -19,6 +22,9 @@ def create
     
   end
 
+  def show 
+    @post = Post.find_by(id: params[:id])
+  end 
 
 
   def edit
```

Comment:
post create and update are done the same way, consider using the same form instead of a different one for each